specification/billing/resource-manager/Microsoft.Billing/stable/2023-04-01/billingSubscription.json

```diff
@@ -400,244 +749,605 @@
+          "type": "string",
+          "readOnly": true
+        },
+        "customerName": {
+          "description": "The ID that uniquely identifies a customer.",
+          "type": "string",
+          "readOnly": true
+        },
         "displayName": {
-          "description": "The name of the subscription.",
-          "readOnly": true,
+          "description": "The name of the billing subscription.",
           "type": "string"
         },
-        "invoiceSectionDisplayName": {
-          "description": "The name of the invoice section to which the subscription is billed.",
-          "readOnly": true,
-          "type": "string"
+        "enrollmentAccountId": {
+          "description": "The enrollment Account ID associated with the subscription. This field is available only for the Enterprise Agreement Type billing accounts.",
+          "type": "string",
+          "readOnly": true
+        },
+        "enrollmentAccountDisplayName": {
+          "description": "The enrollment Account name associated with the subscription. This field is available only for the Enterprise Agreement Type billing accounts.",
+          "type": "string",
+          "readOnly": true
+        },
+        "enrollmentAccountSubscriptionDetails": {
+          "description": "The billing properties that can be modified. Available only for the Enterprise Agreement Type.",
+          "allOf": [
+            {
+              "$ref": "#/definitions/EnrollmentAccountSubscriptionDetails"
+            }
+          ]
         },
         "invoiceSectionId": {
-          "description": "The ID of the invoice section to which the subscription is billed.",
+          "description": "The fully qualified ID that uniquely identifies an invoice section.",
+          "type": "string",
+          "readOnly": true
+        },
+        "invoiceSectionDisplayName": {
+          "description": "The name of the invoice section.",
+          "type": "string",
+          "readOnly": true
+        },
+        "invoiceSectionName": {
+          "description": "The ID that uniquely identifies an invoice section.",
+          "type": "string",
+          "readOnly": true
+        },
+        "nextBillingCycleDetails": {
+          "description": "Billing cycle details of the product.",
+          "allOf": [
+            {
+              "$ref": "#/definitions/NextBillingCycleDetails"
+            }
+          ]
+        },
+        "offerId": {
+          "description": "The offer ID for the subscription. This field is only available for the Microsoft Online Services Program billing accounts.",
+          "type": "string",
+          "readOnly": true
+        },
+        "productCategory": {
+          "description": "Category of the subscription. Possible values include: ReservationOrder, SavingsPlanOrder, AzureSupport, UsageBased, Hardware, SaaS, SeatBased, and Software.",
+          "enum": [
+            "Other",
+            "ReservationOrder",
```

Comment:
I'm worried this is going to be out of sync very soon... For example, what are you going to use for SSC?

Can we either use more general categories (Billing Benefit (for RI/SP/SSC/etc.), SaaS (for SaaS, SeatBased, AzureSupport), etc.) or just remove it and eventually let people look at the resource type in the resourceUri field of each billing subscription? Also, I don't think we can use UsageBased for Cloud Subs anymore, because a Cloud Sub can contain non-usage resources also (e.g. SaaS).

Reply:
@ramkd12 @amberbhargava to comment here